### ISSUE TITLE
'corsWhitelist' should default to empty array

### ISSUE THREAD

**Issue by lukeed** [feathersjs/generator-feathers#90]
I picked `corsWhitelist` in the setup because I wanted to come back to this later.

So, I just pressed enter, sending a blank entry. That entry made its way to here:

```js
/generator-feathers/generators/app/templates/config.default.json:26
    24|     }<% } %>
    25|   }<% } %><% if (cors === 'whitelisted') { %>,
 >> 26|   "corsWhitelist": [<% corsWhitelist.forEach(function(domain, index, list){ %>
    27|     "<%= domain.trim() %>"<% if(index + 1 !== list.length){ %>,<% } %><% }); %>
    28|   ]<% } %>
    29| }

corsWhitelist.forEach is not a function
```

That value should default to an empty array after exploding the comma-string fails.

**Comment by ekryski** [comment]
This is a duplicate of #85. I think this was closed at the time when you reported this (all good!). We've since re-opened it, so I'll close this one. Thanks for reporting!

**Comment by daffl** [comment]
This is fixed in  `v0.6.20` of the generator.
